### DISCUSSION TITLE
What does "🕵️ not run" mean exactly?

### DISCUSSION THREAD

**Discussion by sgabler** [stryker-mutator/stryker-net#1835] [Q&A] [answered]
This may be really dumb, but I'm having problems understanding the report in one area.

My report looks like this:

![image](https://user-images.githubusercontent.com/3625477/145899991-310cdcd3-cb2f-4d2e-a233-05243d122756.png)

Then I click on the first green bubble:

![image](https://user-images.githubusercontent.com/3625477/145900079-f6df0150-868b-4912-9102-07bcb357a1bd.png)

But what does this "🕵️ not run" mean?

![image](https://user-images.githubusercontent.com/3625477/145900099-fc0aa2ab-c99a-49c5-a14f-01a3f742dc2f.png)

I imagine it's an explanation as to why that mutant was killed, but I don't understand it. Maybe I should just go to bed 🤪

**Comment by rouke-broersma** [comment, marked as answer]
Mhm that doesn't seem correct 😅

**Reply by sgabler** [reply]
Oh, that explains it. Maybe I can create an issue from that later.

But anyway, in which cases would "🕵️ not run" be the explanation?

**Reply by rouke-broersma** [reply]
I don't think that should ever be the explanation, there's a bug somewhere in how we fill the description. If you could file a bug report with some information that would be great.

**Reply by sgabler** [reply]
ok i will try to do that in the next couple of days when i find some time. thank you 🙂

**Reply by sgabler** [reply]
@rouke-broersma I created the issue here: #1837

**Comment by psfinaki** [comment]
I think it somehow slipped in together with the static mutants marker, I even thought it's somehow related.

**Reply by rouke-broersma** [reply]
Previously we were not setting the description at all, I guess somewhere we're setting the description to not run for some reason which is getting filled now.
